app/views/profile/index.html.slim

```diff
@@ -1,5 +1,4 @@
-- content_for(:nav) do
-  = render 'shared/nav_auth'
+= shared_nav(greeting: @view_model.header_personalization)
```

Comment:
Do you mean that we should add a spec to test that this method getting called?

The spec for the index view already has a test that checks that the partial is rendered. I also updated the nav partial spec to make sure the `:greeting` local variable was being used.

I'm open to any suggestions of where I can improve the test coverage!